### DISCUSSION TITLE
Isarlink for bulk insert

### DISCUSSION THREAD

**Discussion by WasimMemon** [isar/isar#1212] [Help]
I have bulk insert operation for one table. Let's say for sample project I have bulk insert for student table with links to teachers table or subject table. In student table data I have subject ID and link to subject object. So when I insert subject data then I insert student data in bulk, but my links are not properly initialized I have to re select subjects again. 

I have tried 

students.forEach((element) {element.subject.save();}); 

Please let me know how I can assign links value when inserting data.

**Comment by sumitsharansatsangi** [comment]
Did you tried this `students.forEach((element) {await element.subject.save();});`
Not: save() returns Future.
